crates/rune/src/languageserver/state.rs

```diff
@@ -590,6 +590,9 @@ fn emit_scripts(
             Diagnostic::Warning(e) => {
                 report(build, reporter, e.source_id(), e, to_warning)?;
             }
+            Diagnostic::RuntimeWarning(w) => {
+                todo!()
```

Comment:
My question here is, di you want to show it at all? The actual `report` function is not compatible at all right now...

Reply:
A language server will never execute the code, so will never see runtime diagnostics. Another reason why it's better to collect runtime diagnostics into a separate type, like `RuntimeDiagnostics`.

But at the very least don't leave the panic. This should either emit a warning, error in a recoverable way, or be ignored. We don't want the LS to ever crash.